submissions/idiagheOsaigbovo/CssTask/Registration form/style.css

```diff
@@ -0,0 +1,186 @@
+body
```

Comment:
Please put the curly bracket on the same line as the selector as stated in the CSS section in the styleguide. Please do the same for other selectors in this file.